ngraph/test/util/engine/ie_engines.cpp

```diff
@@ -18,7 +18,6 @@
 
 #include "ngraph/opsets/opset.hpp"
 #include "ngraph/pass/manager.hpp"
-#include "pass/opset1_upgrade.hpp"
```

Comment:
Need to double-check whether IE-Backend is used anywhere or not. If it's not used, let's removed it at all, otherwise, let's not change its functionality.